controllers/informationCenter/form/NewNoteForm.inc.php

```diff
@@ -82,7 +82,7 @@ public function fetch($request, $template = null, $display = false)
         $templateMgr = TemplateManager::getManager($request);
         $noteDao = DAORegistry::getDAO('NoteDAO'); /** @var NoteDAO $noteDao */
         $templateMgr->assign([
-            'notes' => $noteDao->getByAssoc($this->getAssocType(), $this->getAssocId()),
+            'notes' => $noteDao->getByAssoc($this->getAssocType(), $this->getAssocId())->toArray(),
```

Comment:
As above, it would be better to let the template use the `LazyCollection` directly.

Reply:
Changed.